frontend/src/modules/assetManagement/templates/assetManagementModalNewAsset.hbs

```diff
@@ -38,6 +38,46 @@
             <textarea class="asset-description required" name="description" placeholder="{{t 'app.assetdescription'}}">{{ description }}</textarea>
         </div>
 
+        <div class="form-group">
+            <label for="attribution">
+                {{t 'app.assetattribution'}} <span class="req">*</span>
+                <span class="error"></span>
+            </label>
+            <textarea class="asset-attribution required" name="attribution" placeholder="{{t 'app.assetattribution'}}">{{ attribution }}</textarea>
+        </div>
+
+        <div class="form-group">
+            <label for="source">
+                {{t 'app.assetsource'}} <span class="req">*</span>
+                <span class="error"></span>
+            </label>
+            <textarea class="asset-source required" name="source" placeholder="{{t 'app.assetsource'}}">{{ source }}</textarea>
+        </div>
+
+        <div class="form-group">
+            <label for="licence">
+                {{t 'app.assetlicence'}} <span class="req">*</span>
+                <span class="error"></span>
+            </label>
+            <select class="asset-licence required" name="licence" >
+                <option value=""></option>
+                <option value="CC0">CC0</option>
+                <option value="CC-BY">CC-BY</option>
+                <option value="CC-BY-SA">CC-BY-SA</option>
+                <option value="CC-BY-NC">CC-BY-NC</option>
+                <option value="CC-BY-NC-SA">CC-BY-NC-SA</option>
+                <option value="CC-BY-ND">CC-BY-ND</option>
+                <option value="CC-BY-ND">CC-BY-NC-ND</option>
+                <option value="All rights reserved">All rights reserved</option>
+                <option value="Shutterstock standard licence">Shutterstock standard licence</option>
+            </select>
+            <script>
+            $( document ).ready(function() {
+               $('.asset-licence').val('{{ licence }}');
```

Comment:
Can you please try to put this code inside the view that renders the template.

Reply:
I'm not entirely sure how to pull the current value out in the template. I put this here as the template seems to fill in the values for everything else. Help welcome